package.json

```diff
@@ -145,6 +271,7 @@
         "description": "",
         "visible": true,
         "subPermissions": [
+          "ui-finance.fund-budget.edit",
```

Comment:
that looks weird. What is the reason to include whole edit fund-budget permissions here? basically, it's a regression for https://issues.folio.org/browse/UINV-154

Reply:
[Permissions spreadsheet ](https://docs.google.com/spreadsheets/d/1IJnlWvi4k-wFatDNimx9fQ375evHSIIxtGMookHvs-I/edit#gid=1358383889) says: _Allows user to create allocation transactions against budgets. Must include view and edit fund and budget permissions._